shared/common-adapters/floating-menu/index.tsx

```diff
@@ -8,6 +8,7 @@ import * as React from 'react'
 import Overlay from '../overlay'
 import {Position} from '../relative-popup-hoc.types'
 import MenuLayout, {MenuItem, MenuItems} from './menu-layout'
+import ScrollView from '../scroll-view'
```

Comment:
why

Reply:
oops this was from a previous attempt at scrolling